app/main.ts

```diff
@@ -156,9 +145,9 @@ const createWindow = async () => {
 
     initializeIpcHandlers(mainWindow, printWindow, browserView);
 
-    // Remove this if your app does not use auto updates
-    // eslint-disable-next-line
-    new AppUpdater();
+    if (process.env.NODE_ENV === 'production' || process.env.TARGET_NET) {
```

Comment:
If in development mode and a `TARGET_NET` has been specified, why should it then run the auto updater?

Reply:
Doesn't make any sense.. Changed to `&&` instead.